src/vivarium_public_health/results/mortality.py

```diff
@@ -90,14 +90,18 @@ def columns_required(self) -> List[str]:
     # noinspection PyAttributeOutsideInit
     def setup(self, builder: Builder) -> None:
         self.clock = builder.time.clock()
-        self.config = builder.configuration.stratification.mortality
+        self.config = builder.configuration.stratification[self.get_configuration_name()]
```

Comment:
Note: if you have rebased (I can't remember if you have), I recently introduced an instance method on `Component` called `get_configuration()`. You should use that here, but that can wait if you haven't rebased. If you're waiting, can you ticket it, so we don't forget to do it.

Reply:
Won't this not work b/c this is looking for "stratification" at the outer level?

Reply:
You can override the `get_configuration()` method to do anything you want.

Reply:
Ah, yeah ok, I thought you thought it would just work out of the box